guides/moving-to-oss.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "Moving to Coder OSS Beta"
+description: What you need to know about Coder OSS
+---
+
+Coder OSS (also referred to as [Coder OSS](https://github.com/coder/coder)) is
+Coder's open core remote development platform first launched in June 2022. This
+document shares best practices for moving your workflows from Coder v1 to Coder
+OSS.
+
+![Coder OSS
+Dashboard](https://raw.githubusercontent.com/coder/coder/main/docs/images/hero-image.png)
+
+> If you are current Coder v1 customer and to try Coder OSS,
```

Comment:
Grammar:

If you are current Coder v1 customer want to try Coder OSS, [we'd like to hear from you](https://coder.com/contact) to help inform our roadmap and migration strategy.

is this what you meant

Reply:
yep - fixed.